Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -270,4 +281,26 @@ public class HTTPNetworkTransport: NetworkTransport {
     
     return ["query": operation.queryDocument, "variables": operation.variables]
   }
+
+  private func requestMultipartFormData<Operation: GraphQLOperation>(for operation: Operation, files: [GraphQLFile]) -> MultipartFormData {
+	    let formData = MultipartFormData()
+
+	    let fields = requestBody(for: operation)
+	    for (name, data) in fields {
+	      if let data = data as? GraphQLMap {
+	        let data = try! serializationFormat.serialize(value: data)
+	        formData.appendPart(data: data, name: name)
+	      } else if let data = data as? String {
+	        formData.appendPart(string: data, name: name)
+	      } else {
+	        formData.appendPart(string: data.debugDescription, name: name)
+	      }
+	    }
+
+	    for f in files {
+	      formData.appendPart(inputStream: f.inputStream, contentLength: f.contentLength, name: f.fieldName, contentType: f.mimeType, filename: f.originalName)
+	    }
+
+	    return formData
+	  }
```

Comment:
indentation on this is alllll over the place, can you please update it?

Reply:
Indentation on this is still off - We're using 2-space indentation throughout the project, and that's set at the project level. If you select the whole file and do `ctrl-I` then it should indent it properly.